### ISSUE TITLE
MediaPicker Crash on iOS 10

### ISSUE THREAD

**Issue by edecision-plaroche** [xamarin/Xamarin.Mobile#64]
When i cal the method PickPhotoAsync from Xamarin Form, it crash on iOS 10 and not in iOS 9,8.
Here is the output console : 
2016-10-11 19:09:34.848 Projet.iOS[70040:34893132] critical: 
Native stacktrace:

2016-10-11 19:09:34.861 Projet.iOS[70040:34893132] critical:    0   libmonosgen-2.0.dylib               0x059e464d mono_handle_native_sigsegv + 317
2016-10-11 19:09:34.861 Projet.iOS[70040:34893132] critical:    1   libmonosgen-2.0.dylib               0x05a3c321 sigabrt_signal_handler + 161
2016-10-11 19:09:34.861 Projet.iOS[70040:34893132] critical:    2   libsystem_platform.dylib            0x082c079b _sigtramp + 43
2016-10-11 19:09:34.861 Projet.iOS[70040:34893132] critical:    3   ???                                 0xffffffff 0x0 + 4294967295
2016-10-11 19:09:34.862 Projet.iOS[70040:34893132] critical:    4   libsystem_c.dylib                   0x08036137 abort + 141
2016-10-11 19:09:34.862 Projet.iOS[70040:34893132] critical:    5   libsystem_sim_kernel.dylib          0x080f7fd7 __interposition_sim_fallback_abort_with_payload + 0
2016-10-11 19:09:34.862 Projet.iOS[70040:34893132] critical:    6   libsystem_sim_kernel.dylib          0x080f7ff7 __interposition_sim_fallback_abort_with_payload + 32
2016-10-11 19:09:34.862 Projet.iOS[70040:34893132] critical:    7   TCC                                 0x07bc8621 __CRASHING_DUE_TO_PRIVACY_VIOLATION__ + 236
2016-10-11 19:09:34.862 Projet.iOS[70040:34893132] critical:    8   TCC                                 0x07bc8535 **CRASHING_DUE_TO_PRIVACY_VIOLATION** + 0
2016-10-11 19:09:34.862 Projet.iOS[70040:34893132] critical:    9   TCC                                 0x07bcb6b9 __tccd_send_block_invoke + 292
2016-10-11 19:09:34.863 Projet.iOS[70040:34893132] critical:    10  libxpc.dylib                        0x0824879c _xpc_connection_reply_callout + 45
2016-10-11 19:09:34.863 Projet.iOS[70040:34893132] critical:    11  libxpc.dylib                        0x0823fa97 _xpc_connection_call_reply + 32
2016-10-11 19:09:34.863 Projet.iOS[70040:34893132] critical:    12  libdispatch.dylib                   0x07f576ef _dispatch_client_callout + 14
2016-10-11 19:09:34.863 Projet.iOS[70040:34893132] critical:    13  libdispatch.dylib                   0x07f3cc60 _dispatch_queue_override_invoke + 777
2016-10-11 19:09:34.863 Projet.iOS[70040:34893132] critical:    14  libdispatch.dylib                   0x07f3e4bc _dispatch_root_queue_drain + 384
2016-10-11 19:09:34.863 Projet.iOS[70040:34893132] critical:    15  libdispatch.dylib                   0x07f3e2d4 _dispatch_worker_thread3 + 134
2016-10-11 19:09:34.864 Projet.iOS[70040:34893132] critical:    16  libsystem_pthread.dylib             0x082d025c _pthread_wqthread + 1050
2016-10-11 19:09:34.864 Projet.iOS[70040:34893132] critical:    17  libsystem_pthread.dylib             0x082cdf56 start_wqthread + 34
#2016-10-11 19:09:34.864 Projet.iOS[70040:34893132] critical: 

Got a SIGABRT while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries 
# used by your application.

**Comment by Cheesebaron** [comment]
CRASHING_DUE_TO_PRIVACY_VIOLATION

Did you add the needed in use descriptions in your info.plist?

```
<key>NSPhotoLibraryUsageDescription</key>
<string>Describe what photo library is used for</string>
<key>NSCameraUsageDescription</key>
<string>Describe what camera is used for</string>
```

**Comment by edecision-plaroche** [comment]
Yes you are right, i solved the problem yesterday, i am sorry, you can close, so delete if you want my issue.
Many thanks,
Paulin.
